.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels: [Refactor]
+    target-branch: development
```

Maybe we should set a limit on open PR.

```suggestion
    target-branch: development
    open-pull-requests-limit: 1
```

---

I don't think this is needed. We won't have hundreds of updated packages per weeks (one, two ? ) limiting the number of PR would just prevent us from batching those operations.

---

Legitimate :+1: